tests/test_persisted_task_properties.py

```diff
@@ -0,0 +1,28 @@
+"""Tests for validating persisted task properties for different data types."""
+import pytest
+
+import nidaqmx
+import nidaqmx.system
+
+
+class TestPersistedTaskPropertyDataTypes(object):
+    """Contains a collection of pytest tests.
+
+    This validates the property getter,setter and deleter methods for
+    persisted task properties for different data types.
+    """
+
+    @pytest.fixture(scope="class")
+    def voltage_tester_task(self):
+        """Gets the voltage tester task from the persisted tasks in the system."""
+        system = nidaqmx.system.System.local()
+        voltage_tester_task = system.tasks["VoltageTesterTask"]
+        return voltage_tester_task
+
+    def test_boolean_property(self, voltage_tester_task):
```

Comment:
NI's regression testing conventions: https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/20934/Regression-Test-Conventions

Rename to something like test__persisted_task__bool_property__returns_value

Reply:
Updated the method name and remove class names